app/main/views/search.py

```diff
@@ -20,62 +22,135 @@
 
 @main.route('/search/suppliers')
 def supplier_search():
+    cat_list = request.args.getlist('category')
+    response = DataAPIClient().get_roles()
```

Comment:
If the HTTP request fails for any reason, you'll get an exception here, which will turn into an unexplained 500 error to the client.  You'll need to add some error checking.  Personally, I'd make the API client pull its weight a bit more and return an actual list of roles instead of just raw HTTP responses, but since the rest of the client works that way, don't do that in this PR.